src/mbgl/layer/symbol_layer.hpp

```diff
@@ -94,6 +95,8 @@ class SymbolLayer : public StyleLayer {
 
     SymbolLayoutProperties layout;
     SymbolPaintProperties paint;
+
+    SpriteAtlas *layerSpriteAtlas;
```

Comment:
Let's just call this `spriteAtlas`, and set the default value to the regular `SpriteAtlas` (this can happen in `Style::addLayer`). Having it always set will be nice because then no conditional checks are necessary.